src/compile_error_kind.rs

```diff
@@ -138,7 +138,7 @@ pub(crate) enum CompileErrorKind<'src> {
   UnicodeEscapeUnterminated,
   UnknownAliasTarget {
     alias: &'src str,
-    target: &'src str,
+    target: String,
```

Comment:
Can this be a namepath?

```suggestion
    target: Namepath<'src>,
```